.github/CIFLOW.md

```
@@ -0,0 +1,242 @@
+The project follows [GitFlow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) with these branch types:
+
+| Branch Pattern | Base | Purpose |
+|---------------|------|---------|
+| **develop** | — | Main development branch; contains latest development sources |
+| **feature/JNG-xxx_summary** | develop | New features for the next release |
+| **(release/)x.y.z** | develop | Stabilization before a release (`release/` prefix reserved for CI) |
+| **bugfix/JNG-xxx_summary** | release | Bug fixes applied during release testing; merged back to release and develop |
+| **support/JNG-xxx_summary** | release | Minor changes for a previous release; merged back to release |
+| **hotfix/JNG-xxx_summary** | master | Critical fixes applied to both master and develop |
+| **master** | — | Latest released sources |
+
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)